app/models/author.js

```diff
@@ -5,6 +5,7 @@ import { on } from '@ember/object/evented';
 
 export default Publisher.extend({
   name: attr(),
+  biography: attr(),
   books: hasMany(),
```

Comment:
It seems you are missing some details here, like the `attr` type and the `hasMany` relationship model

Reply:
@vladucu Updated changes with `attr` as a string and `hasMany`